pom.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>uk.gov.companieshouse</groupId>
+        <artifactId>companies-house-spring-boot-parent</artifactId>
+        <version>1.3.0</version>
+    </parent>
+    <artifactId>company-documents.web.ch.gov.uk</artifactId>
+    <version>unversioned</version>
+    <name>company-documents.web.ch.gov.uk</name>
+    <description>Web application for handling the retrieval of company documents</description>
+
+    <properties>
+        <java.version>1.8</java.version>
+        <org.mapstruct.version>1.3.0.Final</org.mapstruct.version>
+        <junit-jupiter-engine-version>5.2.0</junit-jupiter-engine-version>
+        <mockito-junit-jupiter-version>2.18.0</mockito-junit-jupiter-version>
+        <hamcrest-all-version>1.3</hamcrest-all-version>
+        <start-class>uk.gov.companieshouse.web.companydocuments.CompanyDocumentsWebApplication</start-class>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-rest</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.junit.vintage</groupId>
+                    <artifactId>junit-vintage-engine</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>${junit-jupiter-engine-version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-junit-jupiter</artifactId>
+            <version>${mockito-junit-jupiter-version}</version>
```

Comment:
I don't think that this is needed, as the property is provided by companies-house-spring-boot-parent. In my copy of Eclipse this leads to an error that points out that we are overriding an already provided version (3.1.0) with our version (2.18.0). We should probably remove this and its counterpart on line 19.